sirepo/feature_config.py

```diff
@@ -82,17 +82,7 @@ def for_sim_type(sim_type):
     c = cfg()
     return pykern.pkcollections.PKDict(
         c[sim_type] if sim_type in c else {}
-    )
-
-
-def sim_common():
-    """Get cfg to use across all simulations. Separate from global cfg
-
-    Returns:
-        dict: common sim config
-    """
-    import pykern.pkcollections
-    return pykern.pkcollections.PKDict(cfg()).sim_common
+    ).pkupdate(c.gui)
```

Comment:
This is an interface change. Is this in use anywhere else?

Reply:
Yes - should simulation_db go back to using a separate call then?

Reply:
No, it looks to me that raydata is not using it right. I'm not sure why it isn't using schema. The raydata use doesn't conflict so should be fine to leave as is.